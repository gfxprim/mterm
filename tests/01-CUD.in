2 5
a\n
\e[B
\e[B
\e[B
\e[B
\e[B
\e[B
\e[B
\e[B
