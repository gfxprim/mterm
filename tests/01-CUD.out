 --
|a |
|  |
|  |
|  |
|  |
 --
size 5x2 cursor 4x0
